src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -282,7 +282,7 @@ public CaseDetails saveCountyCourtJudgment(
 
     private CaseEvent getCCJEvent(CountyCourtJudgmentType countyCourtJudgmentType) {
         if (countyCourtJudgmentType.equals(CountyCourtJudgmentType.ADMISSIONS)) {
```

Comment:
I think we need to also check for DETERMINATION

Reply:
Determination is not in scope of this story.